CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 8.0.0
+
+-   Added: stylelint semantic version policy
```

Comment:
Yes, great change 👍 

It should help us in linking/reference to the v8 changelog.

Maybe we should send out a tweet announcement about said change?